examples/examples_go_test.go

```diff
@@ -12,7 +12,7 @@ func getGoBaseOptions() integration.ProgramTestOptions {
 	base := getBaseOptions()
 	goBase := base.With(integration.ProgramTestOptions{
 		Dependencies: []string{
-			"github.com/pulumi/pulumi-rancher2/sdk/v9",
+			"github.com/pulumi/pulumi-rancher2/sdk/v11",
```

Comment:
Any go examples need to have their import paths updated.

If there are no go examples, we can just delete this file.